src/mkdir.js

```diff
@@ -75,7 +75,7 @@ function _mkdir(options, dirs) {
 
     try {
       if (options.fullpath) {
-        mkdirSyncRecursive(dir);
+        mkdirSyncRecursive(path.resolve(dir));
```

Comment:
It seems like `fs.mkdirSync` normalizes the path passed to it anyway. Is that why `path.resolve` is not needed below?